lib/facebook/messenger/subscriptions.rb

```diff
@@ -8,30 +8,37 @@ module Subscriptions
 
       base_uri 'https://graph.facebook.com/v2.6/me'
 
-      def self.subscribe
-        response = post '/subscribed_apps', format: :json, query: {
-          access_token: Facebook::Messenger.config.access_token
-        }
+      module_function
+
+      def subscribe
+        response = post '/subscribed_apps'
 
         raise_errors(response)
 
         true
       end
 
-      def self.unsubscribe
-        response = delete '/subscribed_apps', format: :json, query: {
-          access_token: Facebook::Messenger.config.access_token
-        }
+      def unsubscribe
+        response = delete '/subscribed_apps'
 
         raise_errors(response)
 
         true
       end
 
-      def self.raise_errors(response)
+      def raise_errors(response)
         raise Error, response['error']['message'] if response.key? 'error'
       end
 
+      def default_options
+        super.merge(
+          format: :json,
```

Comment:
I think we should set this as `format :json` instead, like we've done for `base_uri`.